--- ref
+++ di
@@ -26,7 +26,6 @@
                  Vfront    8 Vsync   8 Vback 1832 Vpol N
     Display Range Limits:
       Monitor ranges (Range Limits Only): 48-120 Hz V, 218-218 kHz H, max dotclock 660 MHz
-    Manufacturer-Specified Display Descriptor (0x00): 00 00 00 00 00 00 00 00 00 00 00 01 00 00 00 00 '................'
     Alphanumeric Data String: 'ATNA40CU06-0 '
   Extension blocks: 1
 Checksum: 0x25
@@ -35,49 +34,14 @@
 
 Block 1, DisplayID Extension Block:
   Version: 2.0
-  Extension Count: 0
   Display Product Primary Use Case: None of the listed primary use cases; generic display
   Product Identification Data Block (0x20), OUI 4C-83-00:
     Product Code: 16797
     Year of Manufacture: 2032
   Display Parameters Data Block (0x21):
-    Image size: 300.0 mm x 190.0 mm
-    Display native pixel format: 2880x1800
-    Scan Orientation: Left to Right, Top to Bottom
-    Luminance Information: Minimum guaranteed value
-    Color Information: CIE 1931
-    Audio Speaker Information: integrated
-    Native Color Chromaticity:
-      Primary #1:  (0.683105, 0.315918)
-      Primary #2:  (0.245117, 0.714111)
-      Primary #3:  (0.139893, 0.043945)
-      White Point: (0.312744, 0.329102)
-    Native Maximum Luminance (Full Coverage): 400.000000 cd/m^2
-    Native Maximum Luminance (10% Rectangular Coverage): 616.000000 cd/m^2
-    Native Minimum Luminance: 0.000500 cd/m^2
-    Native Color Depth: 12 bpc
-    Display Device Technology: Organic LED
-    Native Gamma EOTF: 2.20
   Display Interface Features Data Block:
-    Supported bpc for RGB encoding: 6, 8, 10
-    Supported bpc for YCbCr 4:4:4 encoding: 8, 10
-    Supported bpc for YCbCr 4:2:2 encoding: 8, 10
-    Supported color space and EOTF standard combination 1: DCI-P3, BT.2020/SMPTE ST 2084
   Video Timing Modes Type 7 - Detailed Timings Data Block:
-    DTD:  2880x1800  120.000294 Hz  16:10   218.881 kHz    652.264000 MHz (aspect 16:10, no 3D stereo, preferred)
-               Hfront   32 Hsync   8 Hback   60 Hpol N
-               Vfront    8 Vsync   8 Vback    8 Vpol N
   CTA-861 DisplayID Data Block:
-  Vendor-Specific Data Block (AMD), OUI 00-00-1A:
-    Version: 3
-    Feature Caps: 0x03
-    Minimum Refresh Rate: 48 Hz
-    Maximum Refresh Rate: 120 Hz
-    Flags 1.x: 0x00
-    Flags 2.x: 0xa0
-    Maximum luminance: 116 (616.884 cd/m^2)
-    Minimum luminance: 2 (0.000 cd/m^2)
-    Unknown: 0x60 0x02
   Colorimetry Data Block:
     BT2020RGB
   HDR Static Metadata Data Block:
@@ -88,29 +52,14 @@
       Static metadata type 1
     Desired content max luminance: 116 (616.884 cd/m^2)
     Desired content max frame-average luminance: 96 (400.000 cd/m^2)
-    Desired content min luminance: 2 (0.000 cd/m^2)
-  Checksum: 0x91
+    Desired content min luminance: 1 (0.000 cd/m^2)
 Checksum: 0x90
 
 ----------------
 
-Warnings:
-
-Block 0, Base EDID:
-  Missing Display Product Name.
-Block 1, DisplayID Extension Block:
-  Product Identification Data Block (0x20), OUI 4C-83-00: Unknown OUI 4C-83-00.
-  Colorimetry Data Block: Set the sRGB colorimetry bit to avoid interop issues.
-EDID:
-  Base EDID: Some timings are out of range of the Monitor Ranges:
-    Horizontal Freq: 218.880 - 218.882 kHz (Monitor: 218.000 - 218.000 kHz)
-    Could be due to a Monitor Range off-by-one rounding issue
-
 Failures:
 
 Block 0, Base EDID:
   Display Range Limits: Byte 11 is 0x00 instead of 0x0a.
   Display Range Limits: Bytes 12-17 must be 0x20.
-  Alphanumeric Data String: No newline, but one or more trailing spaces.
-
 EDID conformity: FAIL
